src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3859,4 +3859,7 @@
   <data name="StatusWithColon" xml:space="preserve">
     <value>Status:</value>
   </data>
+  <data name="ShowToolbar" xml:space="preserve">
+    <value>Show toolbar</value>
```

Comment:
Show inner navigation toolbar
Or let's make a description under the title having this text.

Reply:
"inner navigation" is a coding concept, I don't think users are familiar with it.